coding-challenge/src/bank/bank-account.ts

```diff
@@ -10,23 +10,69 @@ export class BankAccount {
         this.accountNumber = uuid();
     }
 
```

Comment:
You left the balance class property public. I don't think it should be a public member because we do not want to be able to edit it from outside this class only via the methods we provide for editing (transfer, withdraw, deposit...)